server/master/src/main/java/org/apache/accumulo/master/tableOps/BulkImport.java

```diff
@@ -177,9 +181,10 @@ private Path createNewBulkDir(VolumeManager fs, String tableId) throws IOExcepti
     }
   }
 
-  private String prepareBulkImport(Master master, final VolumeManager fs, String dir,
-      String tableId) throws Exception {
-    final Path bulkDir = createNewBulkDir(fs, tableId);
+  @VisibleForTesting
```

Comment:
I don't think it's really necessary to use this annotation, especially  in classes that aren't public API. It will be obvious that it's needed for testing, because removing it will affect a test. An inline comment serves the same purpose, but without the dependency on guava.

Reply:
I think I can inline this method.   

I am neutral on using or not using `@VisibleForTesting`.  If there is going to be a change I think it would be best to do it everywhere for consistency.  To me the annotation means that code should only be called by test code and the declaring class.  Theoretically deviations from this could be detected using static analysis.  If such a static analysis tool exists it may be nice to leave it as an annotation.

Reply:
Nevermind about inlining I was thinking of a change I made for #840 to use @VisibleForTesting.  For this code I think I will leave it as is.  I am ok with changing it everywhere in another PR.